packages/flutter/lib/src/material/tabs.dart

```diff
@@ -764,7 +764,7 @@ class TabBar extends StatefulWidget implements PreferredSizeWidget {
     this.dividerHeight,
     this.labelColor,
     this.labelStyle,
-    this.labelPadding,
+    this.labelPadding = EdgeInsets.zero,
```

Comment:
It's documented that kTabLabelPadding is used when this is null (see https://main-api.flutter.dev/flutter/material/TabBar/labelPadding.html), so defaulting it here to zero doesn't seem correct.

This also breaks how the TabBarTheme's labelPadding is applied: https://main-api.flutter.dev/flutter/material/TabBarTheme/labelPadding.html